l3kernel/testfiles-backend/m3graphics001.lvt

```diff
@@ -5,13 +5,11 @@
 \documentclass{minimal}
 \input{regression-test}
 
-\RequirePackage{expl3}
+\RequirePackage{expl3} % As we need to pick up options
 \ExplSyntaxOn
```

Comment:
What really matters here is to load backend driver before using `l3graphics` functions, so using `\sys_load_backend:n {}` or `\sys_ensure_backend:` works too.
```suggestion
\ExplSyntaxOn
\sys_load_backend:n {}
```

Reply:
No, that doesn't work - the way the test is set up uses `\PassOptionsToPackage` here, so currently we need the package loading

Reply:
Sorted by updating the general setup on main then rebasing again with a change here :)